docs/user_guide.rst

```diff
@@ -0,0 +1,11 @@
+.. title:: User guide: contents
```

Comment:
Need to make new menu tab on site for this too (as for new API)

Reply:
Will do